OOP_HW1/src/main/java/Order.java

```diff
@@ -0,0 +1,28 @@
+public class Order {
+    private TicketType type;
+    private int quantity;
+
+    public Order() {
+    }
+
+    public Order(TicketType t, int q) {
+        type = t;
+        quantity = q;
+    }
```

Comment:
Suggest: Have a clearify parameter name.

```suggestion
    public Order(TicketType type, int quantity) {
        this.type = type;
        this.quantity = quantity;
    }
```